demo/addon-config/charts/logging/templates/clusterlogforwarder.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: logging.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
   name: instance
-  namespace: open-cluster-management
+  namespace: open-cluster-management-observability
```

Comment:
I though we were not maintaining the `demo` anymore

Reply:
Was a nit. I clicked request changes by mistake.

And I don't know why this file was selected. I meant the files from `demo`.

Reply:
I agree we need to remove them at some point in time but I suggest to do this separately.